.goreleaser.yaml

```diff
@@ -438,6 +450,78 @@ docker_manifests:
       - ghcr.io/openbao/openbao:{{ .Version }}-alpine-riscv64
       - ghcr.io/openbao/openbao:{{ .Version }}-ubi-amd64
       - ghcr.io/openbao/openbao:{{ .Version }}-ubi-arm64
+  - name_template: hub.docker.com/openbao/openbao:{{ .Version }}
+    skip_push: false
+    image_templates:
+      - hub.docker.com/openbao/openbao:{{ .Version }}-alpine-amd64
+      - hub.docker.com/openbao/openbao:{{ .Version }}-alpine-arm
+      - hub.docker.com/openbao/openbao:{{ .Version }}-alpine-arm64
+      - hub.docker.com/openbao/openbao:{{ .Version }}-alpine-riscv64
+      - hub.docker.com/openbao/openbao:{{ .Version }}-ubi-amd64
```

Comment:
@naphelps For upstream, `hashicorp/vault` was always an alpine based image and `hashicorp/vault-ubi` was always the UBI image; do we want to do that downstream as well? That'd be my preference. I recognize this is outside the scope of this PR, but I missed it earlier.

Reply:
This does seem outside the scope. Maybe we should open another issue to tackle this.